pettingzoo/classic/rps/rps.py

```diff
@@ -26,17 +19,31 @@ def env():
 
 class raw_env(AECEnv):
     """Two-player environment for rock paper scissors.
+    Expandable environment to rock paper scissors lizard spock.
     The observation is simply the last opponent action."""
 
     metadata = {'render.modes': ['human'], "name": "rps_v1"}
 
-    def __init__(self):
+    def __init__(self, additional_action_pairs=0, max_cycles=15):
```

Comment:
`additional_action_pairs` is a bit of an awkward name, and a bit hard to use for someone unfamiliar with generalized rock paper scisors. Since this is user facing, it might be better to have a more intuitive argument, like `num_actions` and then assert that this argument can only have an even number greater than or equal to 3.

Reply:
I totally agree, I'll change it. Thank you for the comment.